app/views/items/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Items</h1>
+<ul class="items">
+  <% @items.each do |item| %>
+    <li>
+    <%= item.name %> - $<%= item.price %> - <%= item.supermarket_name %> - Customers: <%= item.customer_count %>
```

Comment:
YES! Love that you created these methods in the Item model. I know it seems like such a small change, but now the view is just calling a single model method and that model is responsible for gathering and counting that data.